### ISSUE TITLE
F# DotNet Sdk project Add file has no options

### ISSUE THREAD

**Issue by KevinRansom** [Microsoft/visualfsharp#3300]
Selecting Add new file to an F# projects proposes no file types:

![image](https://user-images.githubusercontent.com/5175830/27809990-c636e85e-6009-11e7-942d-92677aad48b0.png)

**Comment by cartermp** [comment]
cc @srivatsn - would this be something on our end, or the project system?

**Comment by srivatsn** [comment]
I just tried this and I can see the templates if the project targets netcoreapp2.0\netstandard2.0. This is because the item template [declares a required version of 2.0](https://github.com/Microsoft/visualfsharp/blob/master/vsintegration/ItemTemplates/CodeFile/Template/CodeFile.vstemplate#L9) The templates are from a time when .NETFramework was the only platform and so the engine just looks at the version component of the TFM. We ran into the same thing with C# templates and we simply removed the RequiredFrameworkVersion property from the templates. The scenario of preventing these from showing up when targeting .NET Framework 1.0 isn't very interesting at this point.
